spacemesh/v2alpha1/tx.proto

```diff
@@ -96,26 +97,26 @@ message TransactionResult {
 }
 
 message TransactionStreamRequest {
-  uint32 start_layer = 1;
-  uint32 end_layer = 2;
-  repeated bytes id = 3;
-  string address = 4;
-  bool watch = 5;
+  uint32 start_layer = 1; // Starting layer for transaction streaming.
```

Comment:
Optional or required?

Reply:
start_layer will be required and end_layer optional